org.jacoco.core.test.validation.groovy/pom.xml

```diff
@@ -26,7 +26,7 @@
 
   <properties>
     <gmaven.version>1.6.2</gmaven.version>
-    <groovy.version>2.5.3</groovy.version>
+    <groovy.version>2.5.6</groovy.version>
```

Comment:
FYI Groovy 2.5.7 was just released, fixing some issues reported during the Hackergarten

Reply:
I know, and new version also works with JDK 13 - https://github.com/jacoco/jacoco/commit/1a476d57546c3f74fb4daa31c04464eafde935d8 ;)